### ISSUE TITLE
using --size and --wait where there are no files greater than size leaves large_taskid None and causes error

### ISSUE THREAD

**Issue by brockpalen** [brockpalen/archivetar#34]
```
archivetar --prefix my-archive --size 100M --wait --destination-dir /flux-support-dd/brockp/NetPIPE/ --verbose
```

If there are no files > 100M there is no large globus but we will have appropriately:

```
INFO:root:Globus Transfer of Oversize files: None
```

But we never check if large_taskid is a UUID and instead send to globus which promptly says it doesn't know about a task with no UUID:

https://github.com/brockpalen/archivetar/blob/4dc2a4476c5eea1d82eb5c2a8a6453fb30dbe861/archivetar/__init__.py#L542-L544

```
        if args.wait and args.size:
            logging.debug("Wait for large_taskid to finish")
            globus.task_wait(large_taskid)
```

This probably needs to be:

```
        if args.wait and args.size and large_taskid:
            logging.debug("Wait for large_taskid to finish")
            globus.task_wait(large_taskid)
```

or maybe can skip checking for `args.size` because `large_taskid`  will only exist if `--size xx` is provided as an option.

```
        if args.wait and large_taskid:
            logging.debug("Wait for large_taskid to finish")
            globus.task_wait(large_taskid)
```

## workaround for now

Don't use `--size`  on any data that doesn't have files > that path.  You can check for any large enough files with:

```
# Print the histogram only
archivetar --prefix my-prefix --dryrun
```

**Comment by brockpalen** [comment]
Example error:

```
DEBUG:root:Wait for large_taskid to finish
ERROR:root:Issue during tar process killing
Traceback (most recent call last):
  File "archivetar", line 21, in <module>
  File "archivetar/__init__.py", line 595, in main
  File "archivetar/__init__.py", line 574, in main
  File "GlobusTransfer/__init__.py", line 182, in task_wait
  File "globus_sdk/services/transfer/client.py", line 1615, in task_wait
  File "globus_sdk/services/transfer/client.py", line 1497, in get_task
  File "globus_sdk/client.py", line 135, in get
  File "globus_sdk/client.py", line 310, in request
globus_sdk.services.transfer.errors.TransferAPIError: ('GET', 'https://transfer.api.globus.org/v0.10/task/None', 'Bearer', 400, 'BadRequest', "Invalid value for path arg 'task_id' of
type 'string_uuid'", 'vzJHxwuza')
[2365704] Failed to execute script 'archivetar' due to unhandled exception!
```

**Comment by brockpalen** [comment]
Fixed in c66d4ba
